### PR TITLE
fix missing image in about/team

### PR DESCRIPTION
**Pull request by Twista**
no description needed

### CODE REVIEW

**Comment by tricoder42** [comment]
Awesome, thank you! We had a few problems with that!

File: static/jade/about/team.jade [hunk 1]
```diff
@@ -87,7 +87,7 @@ block viewport
       p.description Sponsors
       
     .team-member
-      img(src='https://pbs.twimg.com/profile_images/584466697236705281/x70KfVRq.jpg', alt='')
+      img(src='http://pbs.twimg.com/profile_images/656421052692766720/0D5jHooY_400x400.jpg', alt='')
```
Comment (tricoder42) [inline static/jade/about/team.jade]: You should't load external resources from HTTP when using HTTPS... Could you please update it?

Comment (Twista) [inline static/jade/about/team.jade]: yep, of course

**Comment by Twista** [comment]
done, i din't realize web is on https

should be ok now

**Comment by tricoder42** [comment]
Nice, awesome :)

**Pull request merged**